Source/Urho3D/IO/MountPoint.h

```diff
@@ -63,7 +63,7 @@ class URHO3D_API MountPoint : public Object
     virtual const ea::string& GetName() const = 0;
 
     /// Return absolute file name for *existing* identifier in this mount point, if supported.
-    virtual ea::string GetAbsoluteNameFromIdentifier(const FileIdentifier& fileName) const;
+    virtual ea::string GetAbsoluteNameFromIdentifier(const FileIdentifier& fileName, FileMode mode) const;
```

Comment:
I would like to have separate function instead of having weird parameter that contradict the comment